compose.yml

```diff
@@ -9,10 +9,12 @@ suite:
               bundle exec rspec"
   environment:
     - DEBUG_LEVEL=0
-    - ST2_GITURL=${ST2_GITURL}
-    - ST2_GITREV=${ST2_GITREV}
-    - ST2PKG_VERSION=${ST2PKG_VERSION}
-    - ST2PKG_RELEASE=${ST2PKG_RELEASE}
+    ## For the following variables default values are used
+    #  unless explicitly defined:
+    # - ST2_GITURL=
```

Comment:
There are many ways to pass them in, also see `docker-compose run --help` for `-e` option. Unless decided what's the best approach, I suggest to leave them commented, since they bloat output with warnings(